decentraland/unity-renderer
# feat: Moving embedded emotes to addressables

## What does this PR change?

Moves embedded emotes to addressables

The EnsureEssentials test should be present in the Skybox addressable PR as well. Depending on which one goes first, I will merge them together.

The EnsureEssentials test is moved to Editor tests and not Play mode; since we are slowly migrating to that testing pipeline. 

## How to test the changes?

1. Check that embedded emotes appear on the Emotes wheel and the backpack
2. Play ALL the embedded emotes. Check that they are played correctly
3. Move around the world. Walking and idle emotes should be seen on other avatars. Check that they play correctly
4. Enter with a second account and play embedded emotes. Check that they are seen correctly between users

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
